pom.xml

```diff
@@ -249,13 +249,8 @@ under the License.
                       ! https://issues.apache.org/jira/browse/MINVOKER-174
                       -->
                     <setupIncludes>
-                      <setupInclude>setup_jar_module/pom.xml</setupInclude>
-                      <setupInclude>setup_jar_automodule/pom.xml</setupInclude>
-                      <setupInclude>setup_x/pom.xml</setupInclude>
+                      <setupInclude>setup*/pom.xml</setupInclude>
                     </setupIncludes>
-                    <setupExcludes>
-                      <setupExclude>setup_x/**</setupExclude>
-                    </setupExcludes>
```

Comment:
Excludes are not supported by the invoker plugin, so this configuration was a noop